### PR TITLE
Change SearchSession constructor to include user and password parameters

### PR DESCRIPTION
**Pull request by mhl10**

### CODE REVIEW

**Comment by mhl10** [comment]
Hopefully last PR for this branch

File: search/services/index.py [hunk 1]
```diff
@@ -98,8 +98,9 @@ class SearchSession(object):
     # use SSL. Presumably we will use HTTP Auth, or something else.
 
     def __init__(self, host: str, index: str, port: int=9200,
-                 scheme: str='http', **extra) -> None:
-        """
+                 scheme: str='http', user: str=None, password: str=None,
+                 **extra) -> None:
+
         Initialize the connection to Elasticsearch.
```
Comment (erickpeirson) [inline search/services/index.py]: Looks like we're getting a syntax error here?

Comment (erickpeirson) [reply search/services/index.py]: Maybe missing opening quotes: ```"""```

Comment (mhl10) [reply search/services/index.py]: Thanks, I accidentally removed those during the merge

**Pull request merged**